uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -657,7 +658,10 @@ define([
 
               initialCommentsLength = self.request.comments.length;
             }
-          });
+          })
+          .catch(function (error) {
+            self.noEntitlement = ( error === 'No entitlement' );
```

Comment:
Can this message be more descriptive like "You do not have entitlements, kindly get in touch with your manager to get some" etc?